types.ts

```diff
@@ -1,6 +1,9 @@
-import { ServerRequest } from 'https://deno.land/std@0.97.0/http/server.ts'
+import { ServerRequest } from "https://deno.land/std@0.97.0/http/server.ts";
```

Comment:
pls remove semi and double quotes

easiest way is to run `prettier --no-semi --single-quote types.ts`